src/global_logic.js

```diff
@@ -2762,7 +2762,7 @@ module.exports.generateHaisuiData = function (res, arml, summon, prof, chara, st
                                     haisuiBuff[l]["normalKonshin"] += 0.01 * module.exports.calcHaisuiValue("omegaKonshin", amount, slv, remainHP)
                                 }
                                 omegaKonshinIncluded = true;
-                            }
+                            } 
```

Comment:
unnecessary space? and comment below `}//`

Reply:
cleaned.